src/input.rs

```diff
@@ -153,6 +154,8 @@ impl<'a> Input<'a> {
     /// This function will block until the user submits the input. If the user cancels the input,
     /// an error of type `io::ErrorKind::Interrupted` is returned.
     pub fn run(mut self) -> io::Result<String> {
+        let _ctrlc_handle = ctrlc::show_cursor_after_ctrlc(&self.term)?;
```

Comment:
unused variable is necessary here otherwise the handler is dropped immediately

Reply:
yes, this is the correct way to write this. I _think_ you can also add `#[must_use]` to enforce that it isn't dropped immediately but I'm not 100% on that